code/modules/events/false_alarm.dm

```diff
@@ -7,15 +7,21 @@
 	announceWhen	= 0
 	endWhen	=	90
 	var/datum/event_meta/EM
+	var/eventname
 
 
 /datum/event/false_alarm/end()
-	command_announcement.Announce("Error, It appears our previous announcement about a [EM.name] was a sensor glitch. There is no cause for alarm, please return to your stations.", "False Alarm")
-
+	command_announcement.Announce("Error, It appears our previous announcement about [eventname] was a sensor glitch. There is no cause for alarm, please return to your stations.", "False Alarm")
+	if (E)
```

Comment:
E is not attached to the datum, if anywhere, this should be done at the end of the `announce()` proc. Rnow, it's forcing compilation to fail.